src/Doctrine/ORM/CreditMemoRepository.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Doctrine\ORM;
+
+use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
+use Sylius\RefundPlugin\Repository\CreditMemoRepositoryInterface;
+
+class CreditMemoRepository extends EntityRepository implements CreditMemoRepositoryInterface
+{
+    public function findByOrderId(string $orderId): array
+    {
+        return $this->createQueryBuilder('c')
+            ->andWhere('c.order = :orderId')
+            ->addOrderBy('c.issuedAt', 'ASC')
+            ->setParameter('orderId', $orderId)
```

Comment:
In Sylius we have a convention to use `o` instead of `c`:
```suggestion
        return $this->createQueryBuilder('o')
            ->andWhere('o.order = :orderId')
            ->addOrderBy('o.issuedAt', 'ASC')
            ->setParameter('orderId', $orderId)
```